server/server.go

```diff
@@ -591,14 +620,19 @@ func (server HTTPServer) singleRuleEndpoint(writer http.ResponseWriter, request
 		return
 	}
 
-	rule, successful = server.fetchRuleContent(*aggregatorResponse)
+	rule, successful, _ = server.fetchRuleContent(*aggregatorResponse, server.getOSDFlag(request))
```

Comment:
Why do you ignore error here?

Reply:
Because if at least something goes wrong then `successful` will be `false`, including errors. So it's no need to handle it.

Reply:
well, if somebody changes the implementation, then there will be a problem